primo/utils/override_utils.py

```diff
@@ -0,0 +1,293 @@
+#################################################################################
+# PRIMO - The P&A Project Optimizer was produced under the Methane Emissions
+# Reduction Program (MERP) and National Energy Technology Laboratory's (NETL)
+# National Emissions Reduction Initiative (NEMRI).
+#
+# NOTICE. This Software was developed under funding from the U.S. Government
+# and the U.S. Government consequently retains certain rights. As such, the
+# U.S. Government has been granted for itself and others acting on its behalf
+# a paid-up, nonexclusive, irrevocable, worldwide license in the Software to
+# reproduce, distribute copies to the public, prepare derivative works, and
+# perform publicly and display publicly, and to permit others to do so.
+#################################################################################
+
+# Standard libs
+import logging
+from typing import Dict, List, Optional
+
+# Installed libs
+import pandas as pd
+from haversine import Unit, haversine_vector
+
+# User-defined libs
+from primo.data_parser.data_model import OptInputs
+from primo.utils.raise_exception import raise_exception
+
+LOGGER = logging.getLogger(__name__)
+
+
+class Recalculate:
+    """
+    Class for assess whether whether the overridden P&A projects adhere to the constraints
```

Comment:
assess -> assessing and remove the second whether

Reply:
Fixed